src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -454,6 +454,147 @@ internal static string InsertSpacesToString(string original)
             return newText.ToString();
         }
 
+        /// <summary>
+        /// Gets words from text, e.g. ImportFromCSV to ("Import","From","CSV")
+        /// </summary>
+        /// <param name="text">incoming string</param>
+        /// <param name="maxCharacters">Max number of characters per row</param>
+        internal static IEnumerable<string> WrapText(string text, int maxCharacters)
+        {
+            List<string> words = new List<string>();
+            if (string.IsNullOrWhiteSpace(text))
+                return words;
+
+            StringBuilder currentWord = new StringBuilder();
+            currentWord.Append(text[0]);
+            for (int i = 1; i < text.Length; i++)
+            {
+                var curr = text[i];
+                var prev = text[i - 1];
+
+                if (Char.IsUpper(curr) && i != 0 && !Char.IsUpper(prev) && (!curr.Equals(" ")))
+                {
+                    words.Add(currentWord.ToString());
+                    currentWord.Clear();
+                }
+                currentWord.Append(curr);
+            }
+            // Add last word.
+            words.Add(currentWord.ToString());
+            return words;
+        }
+
+        /// <summary>
+        /// Merge rows according max nuber characters per row, e.g. ("Day","Of","Week") => ("Day Of","Week")
+        /// Last row can contain remaining rows, if there is no place to store them,
+        /// e.g. ("Import","From","CSV") => ("Import","From CSV")
+        /// </summary>
+        /// <param name="rows">Incoming rows</param>
+        /// <param name="maxRows">Max number of rows</param>
+        /// <param name="maxCharacters">Max number characters per row</param>
+        internal static IEnumerable<string> ReduceRowCount(IEnumerable<string> rows, int maxRows, int maxCharacters)
```

Comment:
The `maxCharacters` is not required here. The job of this method is to reduce the number of rows, based on the entries inside `rows` parameter. It continuously fold the entry `N - 1` into `N`, until the row count is achieved. For example if the input is `{ "Insert", "Day", "Of", "Week", "Here" }`, and `maxRows == 3` then the iterations would be:

    Iteration 0: { "Insert", "Day", "Of", "Week Here" }
    Iteration 1: { "Insert", "Day", "Of Week Here" }

If `maxRows == 2`, then...

    Iteration 0: { "Insert", "Day", "Of", "Week Here" }
    Iteration 1: { "Insert", "Day", "Of Week Here" }
    Iteration 2: { "Insert", "Day Of Week Here" }